packages/http/src/helpers/httpClient.helper.ts

```diff
@@ -30,8 +31,19 @@ const sanitizeHeaders = (headers: Record<string, any>): Record<string, string> =
   return headers;
 };
 
+const getHostname = (url: string): string => {
+  let hostname = '';
+  try {
```

Comment:
I'd just throw away the old way, since we can just do:
```ts
const getHostname = (url: string): string => {
  const match = url && url.match(/^.+:\/\/([^/]+)/);
  return match ? match[1] : '';
};
```

I'd be glad if you could add a test for this function
```ts
describe('#getHostname', () => {
  it('resolves a hostname from url correctly', () => {
    expect(HttpClientHelper.getHostname('http://google.com')).toBe('google.com');
    expect(HttpClientHelper.getHostname('ws:///some-website.tld/path/path/path?x=1')).toBe('some-website.tld');
    expect(HttpClientHelper.getHostname('file:///xxx/')).toBe('');
  });
  it('returns an empty hostname if url is not valid', () => {
    expect(HttpClientHelper.getHostname('this is not an url')).toBe('');
    expect(HttpClientHelper.getHostname(null as any)).toBe('');
  });
});
```